src/PDS4_CHAN1_IngestLDD.xml

```diff
@@ -161,7 +164,7 @@
         <DD_Value_Domain>
             <enumeration_flag>false</enumeration_flag>
             <value_data_type>ASCII_Short_String_Collapsed</value_data_type>
-            <pattern>([0-9]{1,2}/)?[0-9]{1,10}(:[0-9]{5})?</pattern>
```

Comment:
Is this going to be problem. Are the rules going to be different for Mini-RF and M3?

Reply:
This should just be a more permissive version of the original pattern. It shouldn't reject anything the original pattern accepts. I changed this because the spacecraft clock counts in M3 don't all have five digits after the decimal, presumably due to rounding or something. I haven't looked to see if this is also the case in Mini-RF. If this is a problem, I can change the pattern back and zero-pad the entries in the labels instead.

Reply:
oh, and there's a typo here -- should be {1,5}. i'll fix that and resubmit this.